lib/policy_machine_storage_adapters/active_record.rb

```diff
@@ -55,8 +55,10 @@ def self.buffers
       @buffers ||= {
                      upsert: {},
                      delete: {},
-                     assignments: [],
-                     links: [],
+                     assignments: {},
+                     deassignments: {},
```

Comment:
...does this imply you want to rename `assignments` as `marriages`? Or is the choice just influenced by the wedding you just went to...